src/test/java/core/basesyntax/service/file/impl/DataValidatorImplTest.java

```diff
@@ -0,0 +1,29 @@
+package core.basesyntax.service.file.impl;
+
+import static org.junit.Assert.assertTrue;
+
+import core.basesyntax.service.file.DataValidator;
+import java.util.List;
+import org.junit.Test;
+
+public class DataValidatorImplTest {
+    @Test
+    public void testCorrectInput() {
+        List<String> list1 = List.of("P,Apple,22","S,Banana,10");
+        List<String> list2 = List.of("r,tomato,56","b,Banana,70");
+        DataValidator validator = new DataValidatorImpl();
```

Comment:
let's use @BeforeClass to instantiate validatior once before all tests

Reply:
done